Source/Extensions/Locale+24Hour.swift

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
Since the query for 24 hours is needed for both the timeline and the current time indicator, I pulled out the logic into this extension. I didn't see a better place for it in existing files.